pynestml/transformers/synapse_post_neuron_transformer.py

```diff
@@ -365,6 +365,37 @@ def transform_neuron_synapse_pair_(self, neuron, synapse):
 
         # XXX: TODO
```

Comment:
Is this `TODO` comment required?

Reply:
yes, it refers to "collect all the internal parameters", which is something that should indeed still be done!